### PR TITLE
WIP: Update Solo and SDO data downloaders

### PR DESCRIPTION
**Pull request by nkasmanoff**

### CODE REVIEW

**Review by jejjohnson — APPROVED** [review]
Looks good!

**Comment by nkasmanoff** [comment]
@jejjohnson updated the editors docs. Full disclosure, a lot of these updates were using GitHub copilot, I just told it where to go and confirmed it looked right. 

If the files in editors.py look acceptable along with the changes I made to the first few notebooks, we can merge this.

Unsure if you want to look it over again so I'll hold off until doing a merge!

**Review by jejjohnson — APPROVED** [review]
All good from my end! Merge when you're ready!

**Pull request merged**
